tests/test_check_package_resolved_policy.py

```
@@ -0,0 +1,285 @@
+        "// swift-tools-version: 6.0\n"
+        "import PackageDescription\n\n"
+        "let package = Package(\n"
+        f'    name: "{name}",\n'
+        "    dependencies: [\n"
+        f"{joined}"
+        "    ]\n"
+        ")\n"
+    )
+
+
+def git(repo, *args):
```

Ruff (0.16.0)

[warning] Missing type annotation for `*args`

(ANN002)